%______________________________________________________________________________________________________________________
% @brief    LaTeX2e Resume for Aaron R Quinlan
\documentclass[margin,line]{cv}
\usepackage{url}
\usepackage{enumerate}

%______________________________________________________________________________________________________________________
\begin{document}
\name{\Large Aaron R. Quinlan, Ph.D.}
\begin{resume}
    %__________________________________________________________________________________________________________________
    % Contact Information
    \section{\mysidestyle Contact\\Information}
    Associate Professor with tenure                                                 \hfill (O): 801.585.0406\\%
    Department of Human Genetics                                                    \hfill (Tw): \url{@aaronquinlan}\\%
    Department of Biomedical Informatics                                     \hfill \url{aquinlan@genetics.utah.edu}\\%
    Associate Director of the USTAR Center for Genetic Discovery                      \hfill    \url{quinlanlab.org}\\%
    University of Utah                                                                                       \hfill \\%

    %__________________________________________________________________________________________________________________
    % Education
    \section{\mysidestyle Education}

    \textbf{Boston College}, Chestnut Hill, MA, USA\\
    Ph.D., Biology, 2008\\
    \\
    \textbf{College of William and Mary}, Williamsburg, VA, USA\\
    B.S., Computer Science, 1997


    %__________________________________________________________________________________________________________________
    % Academic Experience
    \section{\mysidestyle Academic Experience}

    \textbf{University of Virginia}, Charlottesville, VA, USA\\
    \textit{Assistant Professor of Public Health Sciences}                              \hfill 2011-2015\\
    Center for Public Health Genomics\\
    \\
    \textbf{University of Virginia}, Charlottesville, VA, USA\\
    \textit{NRSA Postdoctoral Fellow (NHGRI)}                                          \hfill 2008-2011\\


    %__________________________________________________________________________________________________________________
    % Honours and Awards
    \section{\mysidestyle Honors and\\Awards}
    Editorial Board for Oxford Bioinformatics (2016-2018). \\\vspace{1mm}% 
    Organizing Committee for the 2017 Wellcome Trust Genome Informatics meeting. \\\vspace{1mm}%
    Organizing Committee for the 2016 Wellcome Trust Genome Informatics meeting. \\\vspace{1mm}%    
    Program Committee for the 2016 Intelligent Systems for Molecular Biology meeting. \\\vspace{1mm}%
    Finalist for the Benjamin Franklin for Open Access in the Life Sciences. 2016 \\\vspace{1mm}%
    Session Chair for Genome Informatics 2015 at Cold Spring Harbor Laboratories. \\\vspace{1mm}%
    Reviewer for 2015 American Society of Human Genetics Meeting. \\\vspace{1mm}%
    Reviewer for 2014 American Society of Human Genetics Meeting. \\\vspace{1mm}%
    Finalist (90 of \textgreater 1000) for the Gordon and Betty Moore Data Driven Discovery Competition. 2014 \\\vspace{1mm}%
    Moderator for 2014 Genome Reference Consortium Meeting. Cambridge, England. \\\vspace{1mm}%
    Session Chair and reviewer for 2014 American Society of Human Genetics Meeting. \\\vspace{1mm}%
    Session Chair for Genome Informatics 2013 at Cold Spring Harbor Laboratories. \\\vspace{1mm}%
	Finalist for the Benjamin Franklin for Open Access in the Life Sciences. 2013 \\\vspace{1mm}%
    Co-chair, NHLBI Exome Sequence Project Structural Variation Group. 2011-                           \\\vspace{1mm}%
    Instructor for CSHL Advanced Sequencing Technologies Course. 2009-                                 \\\vspace{1mm}%
    Fund for Excellence in Science and Technology Awardee, UVa. (1 of 5). 2011                         \\\vspace{1mm}%
    Ruth L. Kirschstein (NRSA / F32) Postdoctoral Fellowship, NHGRI. 2009-2010                         \\\vspace{1mm}%
    Presidential Fellowship, Boston College. 2004-2007                                                               %


    %__________________________________________________________________________________________________________________
    % Manuscript Review
    \section{\mysidestyle Journal Review}
    \textit{Ad hoc} reviewer for:\\
        \textit{Nature Genetics}, \textit{Nature Methods},\\
        \textit{Nature Biotechnology}, \textit{AJHG}, \\
        \textit{Genome Research}, \textit{Genome Biology}\\
	      \textit{Bioinformatics}, \textit{BMC Bioinformatics},\\
          \textit{Bioessays}, \textit{Genes}, and \\
        \textit{IEEE Transactions On Computational Biology and Bioinformatics}

    Guest editor for \textit{PLoS Computational Biology}

    %__________________________________________________________________________________________________________________
    % Grant Review
    \section{\mysidestyle Grant Review}
    \textbf{NIH}:\\
    NIH GCAT Study Section, Washington, DC (10-Oct-2017 - 12-Oct-2017)\\
    NIH GCAT Study Section, Washington, DC (8-Jun-2016 - 9-Jun-2016)\\
    NIH GCAT Study Section, Chicago, IL (16-Oct-2013 - 17-Oct-2013)\\
    Special Emphasis Panel for Computational Analysis of ENCODE data (U01). (16-May-2012)

    \textbf{Foreign}:\\
    Reviewer for Genome Canada's 2012 Bioinformatics and Computational Biology Competition (9-Dec-2012)\\
    Ad hoc reviewer for Icelandic Research Fund (3-Nov-2015)
    %__________________________________________________________________________________________________________________
    % Publications
    \section{\mysidestyle Publications}
    $\dagger$\textit{ denotes corresponding author}\\
    $\star$\textit{ denotes joint first authors}\\
    $\star$$\star$\textit{ denotes consortium manuscript}


    \textbf{2017} \\

    \begin{list}{*}{}

    \item[50.] Eilbeck K, \textbf{Quinlan AR}$\star$, Yandell M.
    Settling the score: variant prioritization and Mendelian disease. \emph{Nature Reviews Genetics, doi:10.1038/nrg.2017.52}. 2017.

    \item[49.] Pedersen BS, \textbf{Quinlan AR}$\dagger$.
    Who's who? Detecting and resolving sample anomalies in human DNA  sequencing studies with ​peddy. \emph{AJHG. DOI: 10.1016/j.ajhg.2017.01.017}. 2017. 

    \item[48.] Pedersen BS, \textbf{Quinlan AR}$\dagger$.
    cyvcf2: fast, flexible variant analysis with Python. \emph{Bioinformatics. DOI: 10.1093/bioinformatics/btx057}. 2017. 

    \end{list}


    \textbf{2016} \\

    \begin{list}{*}{}

    \item[47.] Pedersen BS, Layer RM, \textbf{Quinlan AR}$\dagger$.
    Vcfanno: fast, flexible annotation of genetic variants. \emph{Genome Biology}. 2016. doi: 10.1186/s13059-016-0973-5

    \item[46.] Ge Y, Onengut-Gumuscu S, \textbf{Quinlan AR}, Mackey AJ, Wright JA, Buckner JH, Habib T, Rich SS, Concannon P.
    Targeted Deep Sequencing in Multiple-Affected Sibships of European Ancestry Identifies Rare Deleterious Variants in PTPN22 that Confer Risk for Type 1 Diabetes. \emph{Diabetes}. 2016. pii: db150322

    \item[45.] Layer RM, Kindlon N, Karczewski K, Exome Aggregation Consortium, \textbf{Quinlan AR}$\dagger$.
    Efficient genotype compression and analysis of large genetic-variation data sets.
    \emph{Nature Methods}. 2015. doi: 10.1038/nmeth.3654

    \end{list}
    

    \textbf{2015} \\

    \begin{list}{*}{}

    \item[44.] Layer R, \textbf{Quinlan AR}$\dagger$.
    A parallel algorithm for N-way interval set intersection.\\
    \emph{In press, IEEE Proceedings}.

    \item[43.] Chiang C, Layer RM, Faust GG, Lindberg MR, Rose DB, Garrison EP, Marth GT, \textbf{Quinlan AR}, Hall IM.
    SpeedSeq: Ultra-fast personal genome analysis and interpretation
    \emph{Nature Methods}. 2015. Oct;12(10):966-8. doi: 10.1038/nmeth.3505.

    \item[42.] Auer PL, Nalls M, Meschia JF, Worrall BB, Longstreth WT Jr, Seshadri S, Kooperberg C, Burger KM, Carlson CS,
    Carty CL, Chen WM, Cupples LA, DeStefano AL, Fornage M, Hardy J, Hsu L, Jackson RD, Jarvik GP, Kim DS,
    Lakshminarayan K, Lange LA, Manichaikul A,  $\star$$\star$\textit{Quinlan AR}, Singleton AB, Thornton TA,
    Nickerson DA, Peters U, Rich SS; National Heart, Lung, and Blood Institute Exome Sequencing Project.
    Rare and Coding Region Genetic Variants Associated With Risk of Ischemic Stroke: The NHLBI Exome Sequence Project.
    \emph{JAMA Neurol}. 2015. May 11. doi: 10.1001/jamaneurol.2015.0582.

    \item[41.] Onengut-Gumuscu S, Chen WM, Burren O, Cooper NJ, \textbf{Quinlan AR}, et al.
    Fine mapping of type 1 diabetes susceptibility loci and evidence for colocalization of causal variants with lymphoid gene enhancers.\\
    \emph{Nature Genetics}. 2015. Apr;47(4):381-6. doi: 10.1038/ng.3245

    \item[40.] Lindberg MR, Hall IM, \textbf{Quinlan AR}$\dagger$.
    Population-based structural variation discovery with Hydra-Multi.
    \emph{Bioinformatics}. 2015. Apr 15;31(8):1286-9. doi: 10.1093/bioinformatics/btu771.

    \item[39.] Church DM, Schneider VA, Steinberg KM, Schatz MC, \textbf{Quinlan AR}, Chin CS, Kitts PA, Aken B, Marth GT,
    Hoffman MM, Herrero J, Mendoza ML, Durbin R, Flicek P.
    Extending reference assembly models.
    \emph{Genome Biol}. 2015. Jan 24;16:13. doi: 10.1186/s13059-015-0587-3.
    
    \item[38.] Do R, Stitziel NO, Won HH, Jørgensen AB, Duga S, Angelica Merlini P, Kiezun A, Farrall M, Goel A, Zuk O, Guella I, Asselta R, Lange LA, Peloso GM, Auer PL; NHLBI Exome Sequencing Project, Girelli D, Martinelli N, Farlow DN, DePristo MA, Roberts R, Stewart AF, Saleheen D, Danesh J, Epstein SE, Sivapalaratnam S, Hovingh GK, Kastelein JJ, Samani NJ, Schunkert H, Erdmann J, Shah SH, Kraus WE, Davies R, Nikpay M, Johansen CT, Wang J, Hegele RA, Hechter E, Marz W, Kleber ME, Huang J, Johnson AD, Li M, Burke GL, Gross M, Liu Y, Assimes TL, Heiss G, Lange EM, Folsom AR, Taylor HA, Olivieri O, Hamsten A, Clarke R, Reilly DF, Yin W, Rivas MA, Donnelly P, Rossouw JE, Psaty BM, Herrington DM, Wilson JG, Rich SS, Bamshad MJ, Tracy RP, Cupples LA, Rader DJ, Reilly MP, Spertus JA, Cresci S, Hartiala J, Tang WH, Hazen SL, Allayee H, Reiner AP, Carlson CS, Kooperberg C, Jackson RD, Boerwinkle E, Lander ES, Schwartz SM, Siscovick DS, McPherson R, Tybjaerg-Hansen A, Abecasis GR, Watkins H, Nickerson DA, Ardissino D, Sunyaev SR, O'Donnell CJ, Altshuler D, Gabriel S, Kathiresan S.
    \emph{Nature}. 2015. Feb 5;518. doi: 10.1038/nature13917.

    \end{list}


    \textbf{2014} \\

    \begin{list}{*}{}

    \item[37.] Dai C, Deng Y, \textbf{Quinlan AR}, Gaskin F, Tsao B, Fu SM.
    Genetics of Systemic Lupus Erythematosus: Immune Responses and End Organ Resistance to Damage.\\
    \emph{Current Opinion in Immunology}. doi:10.1016/j.coi.2014.10.004

    \item[36.] Quick J, \textbf{Quinlan AR}, Loman N.
    A reference bacterial genome dataset generated on the MinION\textsuperscript{TM} portable single-molecule nanopore sequencer.\\
    \emph{GigaScience}. Oct 20;3:22. doi: 10.1186/2047-217X-3-22

    \item[35.] Loman N, \textbf{Quinlan AR}$\dagger$.
    PORETOOLS: a toolkit for working with nanopore sequencing data from Oxford Nanopore.\\
    \emph{Bioinformatics}. doi:10.1093/bioinformatics/btu555, 2014.

    \item[34.] Yi Qiao, \textbf{Quinlan AR}, Amir Jazaeri, Roeland Verhaak, David Wheeler, Gabor Marth.
    SubcloneSeeker: a computational framework for reconstructing tumor clone structure for cancer variant interpretation and prioritization.\\
    \emph{Genome Biology}. Aug 26;15(8):443, 2014.

    \item[33.] \textbf{Quinlan AR}$\dagger$.
    BEDTools: the Swiss-army tool for genome interval arithmetic.\\
    \emph{Curr Protoc Bioinformatics}. doi: 10.1002/0471250953.bi1112s47, 2014.

    \item[32.] Layer R, \textbf{Quinlan AR}$\dagger$, Hall IM$\dagger$.
    LUMPY: A probabilistic framework for sensitive detection of chromosomal rearrangements.\\
    \emph{Genome Biology}. doi:10.1186/gb-2014-15-6-r84, 2014.

    \item[31.] Martin N, Nakamura K, Paila U, Woo J, Brown C, Wright J, Teraoka S, Haghayegh S, McCurdy D, Schneider M, Hu H, \textbf{Quinlan AR}, Gatti R, and Concannon P.
    Homozygous mutation of MTPAP causes cellular radiosensitivity and persistent DNA double strand breaks.\\
    \emph{Cell Death Dis.}. doi: 10.1038/cddis.2014.99, 2014.

    \item[30.] Farber CR, Reich A, Barnes AM, Becerra P, Rauch F, Cabral WA, Bae A, \textbf{Quinlan AR}, Glorieux FH, Clemens TL, and Marini JC.
    A Novel IFITM5 Mutation in Severe Osteogenesis Imperfecta Decreases PEDF Secretion by Osteoblasts.\\
    \emph{J Bone Miner Res}. doi: 10.1002/jbmr.2173, 2014.

    \item[29.] Tabor HK, Auer PL, Jamal SM, Chong JX, Yu JH, Gordon AS, Graubert TA, O'Donnell CJ, Rich SS, Nickerson DA; $\star$$\star$\textit{NHLBI Exome Sequencing Project}, Bamshad MJ.
    Pathogenic variants for Mendelian and complex traits in exomes of 6,517 European and African Americans: implications for the return of incidental results..\\
    \emph{Am J Hum Genet}. doi: 10.1016/j.ajhg.2014.07.006, 2014.

    \item[28.] Lange LA, Hu Y, Zhang H, Xue C, Schmidt EM, Tang ZZ, Bizon C, Lange EM, Smith JD, Turner EH, Jun G, Kang HM, Peloso G, Auer P, Li KP, Flannick J, Zhang J, Fuchsberger C, Gaulton K, Lindgren C, Locke A, Manning A, Sim X, Rivas MA, Holmen OL, Gottesman O, Lu Y, Ruderfer D, Stahl EA, Duan Q, Li Y, Durda P, Jiao S, Isaacs A, Hofman A, Bis JC, Correa A, Griswold ME, Jakobsdottir J, Smith AV, Schreiner PJ, Feitosa MF, Zhang Q, Huffman JE, Crosby J, Wassel CL, Do R, Franceschini N, Martin LW, Robinson JG, Assimes TL, Crosslin DR, Rosenthal EA, Tsai M, Rieder MJ, Farlow DN, Folsom AR, Lumley T, Fox ER, Carlson CS, Peters U, Jackson RD, van Duijn CM, Uitterlinden AG, Levy D, Rotter JI, Taylor HA, Gudnason V Jr, Siscovick DS, Fornage M, Borecki IB, Hayward C, Rudan I, Chen YE, Bottinger EP, Loos RJ, Sætrom P, Hveem K, Boehnke M, Groop L, McCarthy M, Meitinger T, Ballantyne CM, Gabriel SB, O'Donnell CJ, Post WS, North KE, Reiner AP, Boerwinkle E, Psaty BM, Altshuler D, Kathiresan S, Lin DY, Jarvik GP, Cupples LA, Kooperberg C, Wilson JG, Nickerson DA, Abecasis GR, Rich SS, Tracy RP, Willer CJ; $\star$$\star$\textit{NHLBI Grand Opportunity Exome Sequencing Project}.
    Whole-exome sequencing identifies rare and low-frequency coding variants associated with LDL cholesterol.\\
    \emph{Am J Hum Genet}. doi: 10.1016/j.ajhg.2014.01.010, 2014.

    \item[27.] Gordon AS, Tabor HK, Johnson AD, Snively BM, Assimes TL, Auer PL, Ioannidis JP, Peters U, Robinson JG, Sucheston LE, Wang D, Sotoodehnia N, Rotter JI, Psaty BM, Jackson RD, Herrington DM, O'Donnell CJ, Reiner AP, Rich SS, Rieder MJ, Bamshad MJ, Nickerson DA, $\star$$\star$\textit{NHLBI GO Exome Sequencing Project.}.
    Whole-exome sequencing identifies rare and low-frequency coding variants associated with LDL cholesterol.\\
    \emph{Quantifying rare, deleterious variation in 12 human cytochrome P450 drug-metabolism genes in a large-scale exome dataset.}. \emph{Hum Mol Genet}, doi: 10.1093/hmg/ddt588, 2014.

    \end{list}

    \textbf{2013} \\
    \begin{list}{*}{}

    \item[26.] Paila U, Chapman BA, Kirchner R, \textbf{Quinlan AR}$\dagger$.
    GEMINI: Integrative Exploration of Genetic Variation and Genome Annotations.\\
    \emph{PLoS Comput Biol}, 9(7): e1003153. doi:10.1371/journal.pcbi.1003153, 2013.

    \item[25.] Rosenthal EA, Ranchalis J, Crosslin DR, Burt A, Brunzell JD, Motulsky AG, Nickerson DA; $\star$$\star$\textit{NHLBI GO Exome Sequencing Project}, Wijsman EM, Jarvik GP.
    Joint linkage and association analysis with exome sequence data implicates SLC25A40 in hypertriglyceridemia.\\
    \emph{Am J Hum Genet.}, doi: 10.1016/j.ajhg.2013.10.019, 2013.

    \item[24.] Guo DC, Regalado E, Casteel DE, Santos-Cortez RL, Gong L, Kim JJ, Dyack S, Horne SG, Chang G, Jondeau G, Boileau C, Coselli JS, Li Z, Leal SM, Shendure J, Rieder MJ, Bamshad MJ, Nickerson DA; GenTAC Registry Consortium; National Heart, Lung, and $\star$$\star$\textit{Blood Institute Grand Opportunity Exome Sequencing Project}, Kim C, Milewicz DM.
    Recurrent gain-of-function mutation in PRKG1 causes thoracic aortic aneurysms and acute aortic dissections.\\
    \emph{Am J Hum Genet.}, doi: 10.1016/j.ajhg.2013.06.019, 2013.


    \item[23.] O'Connor TD, Kiezun A, Bamshad M, Rich SS, Smith JD, Turner E; NHLBIGO Exome Sequencing Project; ESP Population Genetics, Statistical Analysis Working Group, Leal SM, Akey JM$\star$$\star$. Fine-scale patterns of population stratification confound rare variant association tests.\\
    \emph{PLoS One}, 8(7): e65834. doi:10.1371/journal.pone.0065834, 2013.

    \item[22.] Johnsen JM, Auer PL, Morrison AC, Jiao S, Wei P, Haessler J, Fox K, McGee SR, Smith JD, Carlson CS, Smith N, Boerwinkle E, Kooperberg C, Nickerson DA, Rich SS, Green D, Peters U, Cushman M, Reiner AP; NHLBI Exome Sequencing Project.$\star$$\star$. Common and rare von Willebrand factor (VWF) coding variants, VWF levels, and factor VIII levels in African Americans: the NHLBI Exome Sequencing Project.\\
    \emph{Blood}, 122(4):590-7. doi:10.1182/blood-2013-02-485094, 2013.

    \item[21.] Norton N, Li D, Rampersaud E, Morales A, Martin ER, Zuchner S, Guo S, Gonzalez M, Hedges DJ, Robertson PD, Krumm N, Nickerson DA, Hershberger RE; National Heart, Lung, and Blood Institute GO Exome Sequencing Project and the Exome Sequencing Project Family Studies Project Team.$\star$$\star$. Exome sequencing and genome-wide linkage analysis in 17 families illustrate the complex contribution of TTN truncating variants to dilated cardiomyopathy.\\
    \emph{Circ Cardiovasc Genet.}, 6(2):144-53. doi:10.1161/CIRCGENETICS.111.000062, 2013.

    \item[20.] Malhotra A, Lindberg M, Leibowitz M, Clark R, Faust G, Layer R, \textbf{Quinlan AR}$\dagger$, and Hall IM$\dagger$.
    Breakpoint profiling of 64 cancer genomes reveals numerous complex rearrangements spawned by homology-independent mechanisms. \\
    \emph{Genome Research}, doi:10.1101/gr.143677.112, 2013.

    \item[19.] Fu W, O'Connor TD, Jun G, Kang HM, Abecasis G, Leal SM, Gabriel S, Rieder MJ, Altshuler D, Shendure J, Nickerson DA, Bamshad MJ; NHLBI Exome Sequencing Project, Akey JM.\\
    \emph{Nature.}, 493(7431):216-20. doi:10.1038/nature11690, 2013.

    \item[18.] Layer R, Robins G, Skadron K, \textbf{Quinlan AR}$\dagger$.
    Binary Interval Search (BITS): A Scalable Algorithm for Counting Interval Intersections.\\
    \emph{Bioinformatics}, 29(1):1-7, 2013.

    \item[17.] Boileau C, Guo DC, Hanna N, Regalado ES, Detaint D, Gong L, Varret M, Prakash SK, Li AH, d'Indy H, Braverman AC, Grandchamp B, Kwartler CS, Gouya L, Santos-Cortez RL, Abifadel M, Leal SM, Muti C, Shendure J, Gross MS, Rieder MJ, Vahanian A, Nickerson DA, Michel JB; National Heart, Lung, and Blood Institute (NHLBI) Go Exome Sequencing Project, Jondeau G, Milewicz DM.$\star$$\star$. TGFB2 mutations cause familial thoracic aortic aneurysms and dissections associated with mild systemic features of Marfan syndrome.\\
    \emph{Nature Genetics.}, 44(8):916-21. doi:10.1038/ng.2348, 2013.

    \item[16.] Emond MJ, Louie T, Emerson J, Zhao W, Mathias RA, Knowles MR, Wright FA, Rieder MJ, Tabor HK, Nickerson DA, Barnes KC; National Heart, Lung, and Blood Institute (NHLBI) GO Exome Sequencing Project; Lung GO, Gibson RL, Bamshad MJ.$\star$$\star$. Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis.\\
    \emph{Nature Genetics.}, 44(8):886-9. doi:10.1038/ng.2344, 2013.

    \end{list}

    \textbf{2010 - 2012} \\
    \begin{list}{*}{}

    \item[15.] Krumm N, Sudmant PH, Ko A, O`Roak BJ, NHLBI Exome Sequencing Project,
    \textbf{Quinlan AR}, Nickerson DA, Eichler EE.
    Copy number variation detection and genotyping from exome sequence data.\\
    \textit{Genome Research}, 22(8):1525-32, 2012.

    \item[14.] \textbf{Quinlan AR}, Hall IM.
    Characterizing complex structural variation in germline and somatic genomes.\\
    \textit{Trends in Genetics}, 18:43-53, 2012.

    \item[13.] \textbf{Quinlan AR} and Hall IM.
    Detection and interpretation of genomic structural variation in mammals.\\
    \textit{Methods in Molecular Biology}, 838:225-48, 2012.

    \item[12.] \textbf{Quinlan AR}, Boland MJ, Leibowitz ML, Shumilina S, Pehrson SM, Baldwin KK, Hall IM.
    Paired-end DNA sequencing of induced pluripotent stem cell genomes reveals
    rare structural mutations and retroelement stability.\\
    \textit{Cell Stem Cell}, 9:366-373, 2011.

    \item[11.] Keene KL, \textbf{Quinlan AR}, Hou X, Hall IM, Mychaleckyj, Onengut-Gumuscu S, Concannon P.
    Evidence for two independent associations with type 1 diabetes at the 12q13 locus.\\
    \textit{Genes and Immunity}, 13:66-70, 2011.

    \item[10.] Dale R, Pedersen B, \textbf{Quinlan AR}$\dagger$.
    Pybedtools: a flexible Python library for manipulating genomic datasets and annotations.\\
    \textit{Bioinformatics}, 24:3423-3424, 2011.\\
    \url{packages.python.org/pybedtools/}

    \item[9.] Barnett D, Garrison E, \textbf{Quinlan AR}, Stromberg M, Marth G.
    BamTools: a C++ API and toolkit for analyzing and managing BAM files.\\
    \textit{Bioinformatics}, 12:1691-1692, 2011.\\
    \url{code.google.com/p/bamtools}

    \item[8.] 1000 Genomes Project Consortium..
    A map of human genome variation from population-scale sequencing.\\
    \textit{Nature} 7319:1061-73, 2010.

    \item[7.] \textbf{Quinlan AR} and Hall IM.
    BEDTools: A flexible framework for comparing genomic features.\\
    \textit{Bioinformatics}, 6:841-842, 2010.\\
    \url{code.google.com/p/bedtools}

    \item[6.] \textbf{Quinlan AR}, Clark RA, Sokolova, S, Leibowitx ML, Zhang Y, Hurles ME, Mell JC, Hall IM.
    Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.\\
    \textit{Genome Research}, 20:623-635, 2010.\\
    \url{code.google.com/p/hydra-sv}

    \end{list}


    \textbf{2007 - 2009} \\
    \begin{list}{*}{}

    \item[5.] Sackton, TB, Kulathinal RJ, Bergman CM, Quinlan AR, Dopman E, Marth GT, Hartl DL, Clark AG.
    Population Genomic Inferences from Sparse High-Throughput Sequencing of Two Populations of Drosophila melanogaster.\\
    \textit{Genome Biol Evol}, 1:439-455, 2009.

    \item[4.] Smith D, \textbf{Quinlan AR}$\star$, Peckham HR, \textit{et al}.
    Rapid whole-genome mutational profiling using next-generation sequencing technologies.\\
    \textit{Genome Research}, 18:1638-1642, 2008.

    \item[3.] Hillier LW, Marth GT, \textbf{Quinlan AR}, \textit{et al}.
    Whole Genome Sequencing and SNP Discovery for C. elegans using massively parallel sequencing-by-synthesis.\\
    \textit{Nature Methods}, 5:183-188, 2008.

    \item[2.] \textbf{Quinlan AR}, Stewart D, Stromberg M, Marth GT.
    PyroBayes: Accurate quality scores for 454 Life Science pyrosequences.\\
    \textit{Nature Methods}, 5:179-181, 2008.

    \item[1.] \textbf{Quinlan AR}, Marth GT.
    Primer-site SNPs mask mutations.\\
    \textit{Nature Methods}, 4:192, 2007.

    \end{list}



    %__________________________________________________________________________________________________________________
    % Research Support
    \section{\mysidestyle Active Research Support}

    Project Title: \textit{New algorithms and tools for large-scale genomic analysis.} \\
    PI: Aaron Quinlan \\
    Source: NIH/NHGRI (R01 HG006693-05) \\
    Annual directs: \$331,000 \\
    Period funded: 1-Apr-2016 - 30-Mar-2021 \\

    Project Title: \textit{Monitoring tumor subclonal heterogeneity over time and space.} \\
    MPI: Gabor Marth (Contact) and Aaron Quinlan \\
    Source: NIH/NCI (UCA209999A) \\
    Annual directs: \$150,000 \\
    Period funded: 1-Jul-2016 - 30-Jun-2021 \\

    Project Title: \textit{Software for exploring all forms of genetic variation in any species.} \\
    PI: Aaron Quinlan \\
    Source: NIH/NHGMS (R01GM124355) \\
    Annual directs: \$476,000 \\
    Period funded: 1-July-2017 - 30-Jun-2022 \\
    Status: Pending IRG Review  \\

    Project Title: \textit{The genetic basis of hypersensitivity to ionizing radiation} \\
    PI: Pat Concannon \\
    Source: NIH/NIEHS (R01 ES027121-01) \\
    Annual directs: \$55,000 \\
    Period funded: 1-Oct-2016 - 30-Sep-2021 \\

    Project Title: \textit{A scalable, integrative, multi-omic analysis platform} \\
    PI: Ryan Layer (Contact) and Aaron Quinlan \\
    Source: NIH/NHGRI (K99 HG009532-01) \\
    Annual directs: \$145,200 \\
    Period funded: 1-July-2017 - 30-Jun-2021 \\
    Status: Impact score = 13. Will very likely be funded in Feb 2017 \\  

    Project Title: \textit{Epigenetic engineering to identify and perturb gene regulatory regions involved in cancer etiology and therapy resistance.} \\
    MPI: Jay Gertz (Contact) and Aaron Quinlan \\
    Source: NIH/NHGRI (R01HG009141-01) \\
    Annual directs: \$85,000 \\
    Period funded: 1-July-2017 - 30-Jun-2021 \\
    Status: 5th percentile. Will very likely be funded in April 2017 \\

    \section{\mysidestyle Pending Research Support}

    Project Title: \textit{A powerful web-based discovery platform for rare disease genomics.} \\
    MPI: Daniel MacArthur (Contact) and Aaron Quinlan \\
    Source: NIH/NHGRI (R01HG009141-01) \\
    Annual directs: \$249,700 \\
    Period funded: 1-July-2017 - 30-Jun-2021 \\
    Status: 4th percentile. Will very likely be funded in April 2017 \\

    Project Title: \textit{Epigenetic engineering to identify and perturb gene regulatory regions involved in cancer etiology and therapy resistance.} \\
    MPI: Jay Gertz (Contact) and Aaron Quinlan \\
    Source: NIH/NHGRI (1U01CA217884-01) \\
    Annual directs: \$65,000 \\
    Period funded: 1-July-2017 - 30-Jun-2021 \\
    Status: Pending IRG Review  \\

    \section{\mysidestyle Completed Research Support}

    Project Title: \textit{The genetic basis of simplex autism} \\
    PI: Hilary Coon \\
    Source: Margolis Foundation \\
    Annual directs: \$24,000 \\
    Period funded: 1-Mar-2016 - 30-June-2017 \\

    Project Title: \textit{The genetic basis of simplex autism} \\
    PI: Hilary Coon \\
    Source: Simons Foundation \\
    Annual directs: \$24,000 \\
    Period funded: 1-Mar-2016 - 31-Dec-2016

    Project Title: \textit{A clinical sequencing program to direct treatment of relapsed pediatric cancers.} \\
    PI: Ira Hall and Aaron Quinlan \\
    Source: UVA Health System Research Award  \\
    Period funded: 31-Apr-2013 - 30-Mar-2016

    \vspace{-2mm}
    Project Title: \textit{New oncogenes and regions of genome instability in ovarian cancer.} \\
    PI: Aaron Quinlan \\
    Source: University of Virginia Fund for Excellence in Science and Technology (FEST) \\
    Period funded: 01-May-2011 - 30-Apr-2014

    \vspace{-2mm}
    Project Title: \textit{Defining the genomic architecture of glioblastoma for improved therapy.} \\
    PI: Aaron Quinlan \\
    Source: University of Virginia Cancer Center Pilot Fund\\
    Period funded: 01-Apr-2011 - 31-Dec-2013

    \vspace{-2mm}
    Project Title: \textit{The Role of Copy Number Variants in Type 1 Diabetes.} \\
    PI: Stephen Rich \\
    Source: NIH/NIDDK (DP3 DK085695)\\
    Period funded: 30-Sep-2009 - 30-Jun-2014

    \vspace{-2mm}
    Project Title: \textit{Expression and proteomic characterization of risk loci in type 1 diabetes.} \\
    PI: Stephen Rich \\
    Source: NIH/NIDDK (DP3 DK085678)\\
    Period funded: 25-Sep-2009 - 30-Jun-2014

    Project Title: \textit{Identification of radiation sensitivity alleles by whole exome sequencing.} \\
    PI: Pat Concannon \\
    Co-investigator: Aaron Quinlan \\
    Source: NIH/NIEHS (R21 ES020521-01) \\
    Period funded: 19-Aug-2011 - 31-Jul-2013

    \vspace{-2mm}
    Project Title: \textit{Carry Out Physical Characterization of Contest Samples and Development and Test Bioinformatic Methods for Scoring and Judging Contestant Entries.}\\
    PI: Dean Gaalaas\\
    Source: \\
    Period Archon Genomics X Prize : 27-Mar-2012 - 31-Dec-2012

    \vspace{-2mm}
    Project Title: \textit{Rates and patterns of recurrent structural variation in the mouse genome.}\\
    PI: Aaron Quinlan\\
    Source: NIH/NHGRI (F32 HG005197-02)\\
    Period funded: 01-Aug-2009 - 31-Dec-2010

    %__________________________________________________________________________________________________________________
    % Seminars
    \section{\mysidestyle Lectures}

    $\dagger$\textit{invited lecture}\\
    $\star$\textit{abstract selected lecture}

    \textit{Inferring function from highly constrained coding regions.} \\
    UW Center for Mendelian Genomics; Seattle, WA; \\
    Aug 17, 2017

    $\star$ \textit{Inferring function from highly constrained coding regions.} \\
    BIRS Conference; Banff, Alberta; \\
    Mar 27, 2017

    \textit{Variation deserts and recombination jungles.} \\
    BYU; Provo, UT; \\
    Feb 23, 2017

    \textit{Variation deserts and recombination jungles.} \\
    UCSD (Yeo Lab); San Diego, CA; \\
    Dec 6, 2016

    $\star$ \textit{Direct measurement of the mutagenic impact of recombination through deep genome sequencing of 519 families.} \\
    ASHG 2016; Vancouver, BC; \\
    Oct 22, 2016

    $\star$ \textit{Genetic analysis software for any species.} \\
    PAG 2016; San Diego, CA; \\
    Jan 9, 2016

    $\dagger$ \textit{Making queries of the genome less difficult.} \\
    BIOT 2015; Provo, UT; \\
    Dec 11, 2015

    $\dagger$ \textit{Making queries of the genome less difficult.} \\
    Genome Informatics 2015; Cold Spring Harbor, NY; \\
    Oct 30, 2015

    $\dagger$ \textit{Querying the genome.} \\
    Sanger Institute Seminar Series; Hinxton, England, UK; \\
    Oct 24, 2015 

    \textit{Challenges and (some) solutions for scanning the genome in studies of human disease.} \\
    University of Utah Human Genetics Interest Group; Salt Lake City, UT; \\
    Sep 24, 2015

    $\dagger$ \textit{Variant calling while accounting for
    alternate haplotypes} \\
    Genome Reference Consortium Workshop 2014; Cambridge, England; \\
    Sep 21, 2014

    $\star$ \textit{How does ovarian cancer become resistant to chemotherapy?} \\
    Biology of the Genome 2014; Cold Spring Harbor, NY \\
    May 6-10, 2014

    $\dagger$ \textit{Prioritizing germline and somatic variation in studies of human disease.} \\
    Johns Hopkins School of Medicine. \\
    April 28, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Virginia Biomedical Engineering Seminar Series. \\
    April 4, 2014

    $\dagger$ \textit{Algorithms for chromosomal rearrangement detection and DNA classification.} \\
    Cold Spring Harbor Laboraties Quantitative Biology Seminar Series. \\
    March 19, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Florida Genetics Institute. \\
    March 12, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Utah Department of Human Genetics. \\
    February 26, 2014

    $\star$ \textit{Disease variant interpretation and prioritization with GEMINI.} \\
    Genome Informatics 2013; Cold Spring Harbor, NY \\
    October 30, 2013

    $\star$ \textit{Mining genomic feature sets and identifying significant biological relationships with BedTools2.} \\
    American Society of Human Genetics; Boston, MA \\
    October 22, 2013

    $\star$ \textit{Disease variant interpretation and prioritization with GEMINI.} \\
    Beyond the Genome 2013; San Francisco, CA \\
    October 3, 2013

    $\dagger$ \textit{Detection and characterization of complex rearrangements in
	tumor genomes.} \\
    BioConductor 2013; Seattle, WA \\
    July 18, 2013

    $\star$ \textit{Exploring disease genetics among thousands of human genomes
	with GEMINI.} \\
    SciPy 2013; Austin, TX \\
    June 26, 2013

    \textit{Computational Genomics.} \\
    Big Data Summit 2 at UVa; Charlottesville, VA \\
    May 14, 2013

    \textit{Exploring genetic variation with a tour guide.} \\
    International Stroke Genetics Consortium Meeting; Charlottesville, VA \\
    April 25, 2013

    $\star$ \textit{LUMPY: A probabilistic framework for SV discovery.} \\
    Advances in Genome Biology and Technology (AGBT); Marco Island, FL \\
    February 22, 2013

    \textit{Mining the genome.} \\
    UVa. Center for Public Health Genomics Genome Sciences Seminar Series \\
    November 28, 2012

    $\dagger$ \textit{Mining the structure and function of the genome.} \\
    Penn State, Dept. of Biochemistry and Molecular Biology \\
    November 12, 2012; Host: Anton Nekreutenko

    \textit{Exploring high-dimension genomic data.} \\
    Cold Spring Harbor Laboratories Advanced Sequencing Technologies Course \\
    October 22, 2012

    \textit{Towards a map of structural variation in the Exome Sequencing Project.} \\
    NHLBI Exome Sequencing Project In-Person Meeting \\
    March 28, 2012

    $\dagger$ \textit{Exploring the origin and extent of structural variation in human genomes.} \\
    Dean’s New Faculty Seminar Series, University of Virginia School of Medicine \\
    Jan 19, 2012

    \textit{ESP Structural Variation Project Group: goals, initial results, and future work.} \\
    NHLBI Exome Sequencing Project In-Person Meeting \\
    June 9, 2011

    $\star$ \textit{Large-Scale Characterization of SV Breakpoints in Cancer.} \\
    Keystone Symposium on The Functional Impact of Structural Variation \\
    Jan. 11, 2011

    $\star$ \textit{Efficient discovery of structural instability in repetitive regions of mammalian genomes.} \\
    Advances in Genome Biology and Technology \\
    Feb. 2009

    \textit{Approaches to rare allele discovery: More samples or more depth per sample? } \\
    1000 Genomes Analysis Meeting, Cold Spring Harbor Laboratories \\
    May 2008.

    %__________________________________________________________________________________________________________________
    % Academic Service
    %__________________________________________________________________________________________________________________
    \section{\mysidestyle Academic Service}

    Health Sciences Research Council, University of Utah. \\
    2017 - 2019.

    Utah Genome Project Ambassador Program, University of Utah. \\
    2017 - 2019.

    Molecular Biology Admissions Committee, University of Utah. \\
    2015 - 2017.

    Judge for BIMS Graduate Student Poster Session. \\
    19-Apr-2013.

    Data Management Committee. Organized by Rick Horwitz, VPR.\\
    12-Apr-2013.

    Big Data Analytics Committee. Organized by Don Brown, Systems Engineering.\\
    29-Mar-2013.

    Member of Center for Public Health Genomics Executive Committee.\\
    01-Aug-2012 - Present.

    \vspace{-2mm}
    Member of the Univ. of Virginia Bioinformatics Core Advisory Committee.\\
    01-Nov-2011 - Present.

    \vspace{-2mm}
    Served on the Univ. of Virginia Bioinformatics Core Director Search Committee. \\
    Summer 2011.
    %__________________________________________________________________________



    %__________________________________________________________________________
    % Teaching
    %__________________________________________________________________________
    \section{\mysidestyle Teaching}

    Applied Computational Genomics. University of Utah. Spring 2017. \\

    Applied Genomics II (PH TX 7778-001). University of Utah, Spring 2016. \\

    Evolutionary Genetics and Genomics (HGEN 6092). University of Utah, Spring 2016. \\

    Advanced Genomics Journal Club (BMI 7010-003/HGEN 6810-001). University of Utah, Spring 2016. \\

    Faculty for CSHL Advanced Sequencing Technologies Course, 2009 - 2017\\
    \emph{- http://meetings.cshl.edu/courses/2013/c-seqtech13.shtml}

    \vspace{-2mm}
    Faculty for the University of Washington's Center for Mendelian Genetics Workshop. 2014 - 2017

    \vspace{-2mm}
    Faculty for Canadian Bioinformatics Workshops. 2012, 2013, 2014 \\
    \emph{- http://bioinformatics.ca/workshops/faculty}

    \vspace{-2mm}
    Guest Lecturer for first year Computer Science graduate student core course (CS 6190), Fall 2013. Taught by Kevin Skadron.

    \vspace{-2mm}
    Lecturer for first year graduate student core course (BIMS 6000), Fall 2013 \\
    \emph{- Genomics Lecture and practical session} - 04-Sep-2013 \\
    \emph{- Research article discussions} - 5-Sep-2013

    \vspace{-2mm}
    Guest Lecturer for undergraduate and graduate Biomedical Engineering course (BME 4806, 7806; Prof. Brent French), \\
    \emph{April 8, 2013} \\
    \emph{April 14, 2014} \\


    Guest Lecturer for graduate course in genomics (BIOCH 5080), Spring 2013. \\
	\emph{March 20, March 22, April 29}

    Lecturer for first year medical student curriculum, Fall 2012. \\
	\emph{- Genomics research article discussions} - 09-Oct-2012

    \vspace{-2mm}
    Lecturer for first year graduate student core course (BIMS 6000), Fall 2012 \\
	\emph{- Genomics Lecture and practical session} - 07-Sep-2012 \\
	\emph{- Research article discussions} - 10-Sep-2012, 11-Sep-2012

    \vspace{-2mm}
    Guest Lecturer for graduate course in genomics (BIOCH 5080), 15-Feb-2011
    %__________________________________________________________________________



    %__________________________________________________________________________
    % Mentorship
    %__________________________________________________________________________
    \section{\mysidestyle Mentorship}
    \textbf{Current} \\
    James Havrilla (Ph.D. candidate, U. of Utah, Human Genetics, started January 2014) \\
    Thomas Sasani (Ph.D. candidate, U. of Utah, Human Genetics, started April 2016) \\
    Stephanie Kravitz (Ph.D. candidate, U. of Utah, Human Genetics, started April 2017) \\
    Jonathan Belyeu (Ph.D. candidate, U. of Utah, Human Genetics, started April 2017) \\


    \textbf{Former} \\
    John Kubinski (Undergraduate Biology Senior Thesis candidate)  \\
    Ryan Layer (Ph.D. candidate, Computer Science, graduated 2014) \\
    Phanwadee Sinthong (Undergraduate Computer Science researcher) \\

    %__________________________________________________________________________


    %__________________________________________________________________________
    % Thesis Committees
    %__________________________________________________________________________
    \section{\mysidestyle Thesis Committees}
    \textbf{Current} \\
    Edwin Lin (MD/Ph.D. candidate, U. of Utah, Human Genetics)\\
    Nicole Russell (MD/Ph.D. candidate, U. of Utah, Human Genetics)\\
    Kristi Russell (Ph.D. candidate, U. of Utah, Human Genetics)\\
    Julia Carleton (Ph.D. candidate, U. of Utah, Oncological Sciences)\\
    Eric Bogenschutz (Ph.D. candidate, U. of Utah, Human Genetics)\\
    Rachel Cosby (Ph.D. candidate, U. of Utah, Human Genetics)\\
    Cassandra Garner (Ph.D. candidate, U. of Utah, Human Genetics)\\
    Samuel Brady (M.S. candidate, U. of Utah, Biomedical Informatics)\\
    Paris Vail (M.S. candidate, U. of Utah, Biomedical Informatics)

    \textbf{Former} \\
    Michael Lindberg (Ph.D. candidate, Biochemistry and Molecular Genetics, qualified 2012)\\
    Lauren Mills (Ph.D., candidate Biochemistry and Molecular Genetics)\\
	Johnny Gan (Ph.D., candidate Systems Engineering)
    %__________________________________________________________________________



%______________________________________________________________________________
\end{resume}
\end{document}


%______________________________________________________________________________
% EOF
