%______________________________________________________________________________________________________________________
% @brief    LaTeX2e Resume for Aaron R Quinlan
\documentclass[margin,line]{cv}
\usepackage{url}
\usepackage{enumerate}

%______________________________________________________________________________________________________________________
\begin{document}
\name{\Large Aaron R. Quinlan, Ph.D.}
\begin{resume}
    %__________________________________________________________________________________________________________________
    % Contact Information
    \section{\mysidestyle Contact\\Information}
    Assistant Professor                                                                     \hfill (O): 434.243.1669\\%
    Department of Public Health Sciences                                                    \hfill (F): 434.924.1312\\%
    Center for Public Health Genomics                                                \hfill \url{arq5x@virginia.edu}\\%
    Department of Biochemistry and Molecular Genetics                                 \hfill    \url{quinlanlab.org}\\%
    Department of Computer Science                                                                            \hfill\\%
    University of Virginia                                                                                   \hfill \\%

    %__________________________________________________________________________________________________________________
    % Research Interests
    \section{\mysidestyle Research\\Interests}
	
    Human genome interpretation;
    Application of genomics to clinical care; 
    Chromosome stability and somatic genome evolution;
    Algorithm and genomics software development;
    Cancer genomics;
    Nucleotide repeat disorders; 
    Population genomics; 
    Genetics of complex disease 

    %__________________________________________________________________________________________________________________
    % Education
    \section{\mysidestyle Education}

    \textbf{Boston College}, Chestnut Hill, MA, USA\\
    Ph.D., Biology, 2008\\
    \\
    \textbf{College of William and Mary}, Williamsburg, VA, USA\\
    B.S., Computer Science, 1997


    %__________________________________________________________________________________________________________________
    % Academic Experience
    \section{\mysidestyle Academic Experience}

    \textbf{University of Virginia}, Charlottesville, VA, USA\\
    \textit{Assistant Professor of Public Health Sciences}                              \hfill 2011-\\
    Center for Public Health Genomics\\
    \\
    \textbf{University of Virginia}, Charlottesville, VA, USA\\
    \textit{NRSA Postdoctoral Fellow (NHGRI)}                                          \hfill 2008-2011\\


    %__________________________________________________________________________________________________________________
    % Honours and Awards
    \section{\mysidestyle Honors and\\Awards} 

    Finalist (90 of \textgreater 1000) for the Gordon and Betty Moore Data Driven Discovery Competition. 2014 \\\vspace{1mm}%
    Moderator for 2014 Genome Reference Consortium Meeting. Cambridge, England. \\\vspace{1mm}% 
    Session Chair for 2014 American Society of Human Genetics Meeting. \\\vspace{1mm}% 
    Session Chair for Genome Informatics 2013 at Cold Spring Harbor Laboratories. \\\vspace{1mm}%    
	Finalist for the Benjamin Franklin for Open Access in the Life Sciences. 2013 \\\vspace{1mm}%
    Co-chair, NHLBI Exome Sequence Project Structural Variation Group. 2011-                           \\\vspace{1mm}%
    Instructor for CSHL Advanced Sequencing Technologies Course. 2009-                                 \\\vspace{1mm}%
    Fund for Excellence in Science and Technology Awardee, UVa. (1 of 5). 2011                         \\\vspace{1mm}%
    Ruth L. Kirschstein (NRSA / F32) Postdoctoral Fellowship, NHGRI. 2009-2010                         \\\vspace{1mm}%
    Presidential Fellowship, Boston College. 2004-2007                                                               %
    
    

    %__________________________________________________________________________________________________________________
    % Manuscript Review
    \section{\mysidestyle Journal Review} 
    \textit{Ad hoc} reviewer for:\\ 
        \textit{Nature Methods}, \textit{AJHG}, \textit{Genome Research}, \textit{Genome Biology}\\
	    \textit{Bioinformatics}, \textit{BMC Bioinformatics}, \textit{Bioessays}, \textit{Genes}, and \\
        \textit{IEEE Transactions On Computational Biology and Bioinformatics}
    
    Guest editor for \textit{PLoS Computational Biology}

    %__________________________________________________________________________________________________________________
    % Grant Review
    \section{\mysidestyle Grant Review} 
    \textbf{NIH}:
    \\
    Special Emphasis Panel for Computational Analysis of ENCODE data (U01). (16-May-2012)\\%
    NIH GCAT Study Section, Chicago, IL (16-Oct-2013 - 17-Oct-2013)%
    \\
    \\
    \textbf{Genome Canada}:
    \\
    Reviewer for 2012 Bioinformatics and Computational Biology Competition (9-Dec-2012)
    
    %__________________________________________________________________________________________________________________
    % Publications
    \section{\mysidestyle Publications}
    $\dagger$\textit{ denotes corresponding author}\\
    $\star$\textit{ denotes joint first authors}\\
    $\star$$\star$\textit{ denotes consortium manuscript}

    \textbf{2014} \\

    \begin{list}{*}{}

    \item[36.] Lindberg M, Hall IM, \textbf{Quinlan AR}$\dagger$.
    Population-based structural variation discovery with Hydra-Multi.\\
    \emph{Under review}.

    \item[35.] Layer R, \textbf{Quinlan AR}$\dagger$.
    A parallel algorithm for N-way interval set intersection.\\
    \emph{Under review}.

    \item[34.] Ge Y, Onengut-Gumuscu S, et al.
    Targeted Deep Sequencing in Multiple-Affected Sibships Identifies Rare Variants Contributing to Risk of Type 1 Diabetes.\\
    \emph{Under review}.

    \item[33.] Onengut-Gumuscu S, Chen WM, et al.
    Comparison of type 1 diabetes loci with 15 other immune diseases and evidence for co-localisation of diabetes causal variants with lymphoid gene enhancers.\\
    \emph{Under review}.

    \item[32.] Dai C, Deng Y, \textbf{Quinlan AR}, Gaskin F, Tsao B, Fu SM.
    Genetics of Systemic Lupus Erythematosus: Immune Responses and End Organ Resistance to Damage.\\
    \emph{In press}.

    \item[31.] Quick J, \textbf{Quinlan AR}, Loman N.
    A reference bacterial genome dataset generated on the MinION\textsuperscript{TM} portable single-molecule nanopore sequencer.\\
    \emph{In press}.

    \item[30.] Loman N, \textbf{Quinlan AR}$\dagger$.
    PORETOOLS: a toolkit for working with nanopore sequencing data from Oxford Nanopore.\\
    \emph{Bioinformatics}. doi:10.1093/bioinformatics/btu555, 2014.

    \item[29.] Yi Qiao, \textbf{Quinlan AR}, Amir Jazaeri, Roeland Verhaak, David Wheeler, Gabor Marth.
    SubcloneSeeker: a computational framework for reconstructing tumor clone structure for cancer variant interpretation and prioritization.\\
    \emph{Genome Biology}. Aug 26;15(8):443, 2014.

    \item[28.] \textbf{Quinlan AR}$\dagger$.
    BEDTools: the Swiss-army tool for genome interval arithmetic.\\
    \emph{Curr Protoc Bioinformatics}. doi: 10.1002/0471250953.bi1112s47, 2014.

    \item[27.] Layer R, \textbf{Quinlan AR}$\dagger$, Hall IM$\dagger$.
    LUMPY: A probabilistic framework for sensitive detection of chromosomal rearrangements.\\
    \emph{Genome Biology}. doi:10.1186/gb-2014-15-6-r84, 2014.

    \item[26.] Martin N, Nakamura K, Paila U, Woo J, Brown C, Wright J, Teraoka S, Haghayegh S, McCurdy D, Schneider M, Hu H, \textbf{Quinlan AR}, Gatti R, and Concannon P.
    Homozygous mutation of MTPAP causes cellular radiosensitivity and persistent DNA double strand breaks.\\
    \emph{Cell Death Dis.}. doi: 10.1038/cddis.2014.99, 2014.

    \item[25.] Farber CR, Reich A, Barnes AM, Becerra P, Rauch F, Cabral WA, Bae A, \textbf{Quinlan AR}, Glorieux FH, Clemens TL, and Marini JC.
    A Novel IFITM5 Mutation in Severe Osteogenesis Imperfecta Decreases PEDF Secretion by Osteoblasts.\\
    \emph{J Bone Miner Res}. doi: 10.1002/jbmr.2173, 2014.
    
    \end{list}

    \textbf{2013} \\
    \begin{list}{*}{}
    
    \item[24.] Paila U, Chapman BA, Kirchner R, \textbf{Quinlan AR}$\dagger$.
    GEMINI: Integrative Exploration of Genetic Variation and Genome Annotations.\\
    \emph{PLoS Comput Biol}, 9(7): e1003153. doi:10.1371/journal.pcbi.1003153, 2013.

    \item[23.] O'Connor TD, Kiezun A, Bamshad M, Rich SS, Smith JD, Turner E; NHLBIGO Exome Sequencing Project; ESP Population Genetics, Statistical Analysis Working Group, Leal SM, Akey JM$\star$$\star$. Fine-scale patterns of population stratification confound rare variant association tests.\\
    \emph{PLoS One}, 8(7): e65834. doi:10.1371/journal.pone.0065834, 2013.

    \item[22.] Johnsen JM, Auer PL, Morrison AC, Jiao S, Wei P, Haessler J, Fox K, McGee SR, Smith JD, Carlson CS, Smith N, Boerwinkle E, Kooperberg C, Nickerson DA, Rich SS, Green D, Peters U, Cushman M, Reiner AP; NHLBI Exome Sequencing Project.$\star$$\star$. Common and rare von Willebrand factor (VWF) coding variants, VWF levels, and factor VIII levels in African Americans: the NHLBI Exome Sequencing Project.\\
    \emph{Blood}, 122(4):590-7. doi:10.1182/blood-2013-02-485094, 2013.

    \item[21.] Norton N, Li D, Rampersaud E, Morales A, Martin ER, Zuchner S, Guo S, Gonzalez M, Hedges DJ, Robertson PD, Krumm N, Nickerson DA, Hershberger RE; National Heart, Lung, and Blood Institute GO Exome Sequencing Project and the Exome Sequencing Project Family Studies Project Team.$\star$$\star$. Exome sequencing and genome-wide linkage analysis in 17 families illustrate the complex contribution of TTN truncating variants to dilated cardiomyopathy.\\
    \emph{Circ Cardiovasc Genet.}, 6(2):144-53. doi:10.1161/CIRCGENETICS.111.000062, 2013.
	
    \item[20.] Malhotra A, Lindberg M, Leibowitz M, Clark R, Faust G, Layer R, \textbf{Quinlan AR}$\dagger$, and Hall IM$\dagger$.
    Breakpoint profiling of 64 cancer genomes reveals numerous complex rearrangements spawned by homology-independent mechanisms. \\
    \emph{Genome Research}, doi:10.1101/gr.143677.112, 2013.

    \item[19.] Fu W, O'Connor TD, Jun G, Kang HM, Abecasis G, Leal SM, Gabriel S, Rieder MJ, Altshuler D, Shendure J, Nickerson DA, Bamshad MJ; NHLBI Exome Sequencing Project, Akey JM.\\
    \emph{Nature.}, 493(7431):216-20. doi:10.1038/nature11690, 2013.
	
    \item[18.] Layer R, Robins G, Skadron K, \textbf{Quinlan AR}$\dagger$. 
    Binary Interval Search (BITS): A Scalable Algorithm for Counting Interval Intersections.\\
    \emph{Bioinformatics}, 29(1):1-7, 2013.
	
    \item[17.] Boileau C, Guo DC, Hanna N, Regalado ES, Detaint D, Gong L, Varret M, Prakash SK, Li AH, d'Indy H, Braverman AC, Grandchamp B, Kwartler CS, Gouya L, Santos-Cortez RL, Abifadel M, Leal SM, Muti C, Shendure J, Gross MS, Rieder MJ, Vahanian A, Nickerson DA, Michel JB; National Heart, Lung, and Blood Institute (NHLBI) Go Exome Sequencing Project, Jondeau G, Milewicz DM.$\star$$\star$. TGFB2 mutations cause familial thoracic aortic aneurysms and dissections associated with mild systemic features of Marfan syndrome.\\
    \emph{Nature Genetics.}, 44(8):916-21. doi:10.1038/ng.2348, 2013.

    \item[16.] Emond MJ, Louie T, Emerson J, Zhao W, Mathias RA, Knowles MR, Wright FA, Rieder MJ, Tabor HK, Nickerson DA, Barnes KC; National Heart, Lung, and Blood Institute (NHLBI) GO Exome Sequencing Project; Lung GO, Gibson RL, Bamshad MJ.$\star$$\star$. Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis.\\
    \emph{Nature Genetics.}, 44(8):886-9. doi:10.1038/ng.2344, 2013.

    \end{list}

    \textbf{2010 - 2012} \\
    \begin{list}{*}{}

    \item[15.] Krumm N, Sudmant PH, Ko A, O`Roak BJ, NHLBI Exome Sequencing Project, 
    \textbf{Quinlan AR}, Nickerson DA, Eichler EE. 
    Copy number variation detection and genotyping from exome sequence data.\\
    \textit{Genome Research}, 22(8):1525-32, 2012.

    \item[14.] \textbf{Quinlan AR}, Hall IM. 
    Characterizing complex structural variation in germline and somatic genomes.\\
    \textit{Trends in Genetics}, 18:43-53, 2012.

    \item[13.] \textbf{Quinlan AR} and Hall IM. 
    Detection and interpretation of genomic structural variation in mammals.\\
    \textit{Methods in Molecular Biology}, 838:225-48, 2012.
   
    \item[12.] \textbf{Quinlan AR}, Boland MJ, Leibowitz ML, Shumilina S, Pehrson SM, Baldwin KK, Hall IM. 
    Paired-end DNA sequencing of induced pluripotent stem cell genomes reveals 
    rare structural mutations and retroelement stability.\\
    \textit{Cell Stem Cell}, 9:366-373, 2011.
    
    \item[11.] Keene KL, \textbf{Quinlan AR}, Hou X, Hall IM, Mychaleckyj, Onengut-Gumuscu S, Concannon P.
    Evidence for two independent associations with type 1 diabetes at the 12q13 locus.\\
    \textit{Genes and Immunity}, 13:66-70, 2011.

    \item[10.] Dale R, Pedersen B, \textbf{Quinlan AR}$\dagger$. 
    Pybedtools: a flexible Python library for manipulating genomic datasets and annotations.\\
    \textit{Bioinformatics}, 24:3423-3424, 2011.\\
    \url{packages.python.org/pybedtools/}
    
    \item[9.] Barnett D, Garrison E, \textbf{Quinlan AR}, Stromberg M, Marth G.
    BamTools: a C++ API and toolkit for analyzing and managing BAM files.\\
    \textit{Bioinformatics}, 12:1691-1692, 2011.\\
    \url{code.google.com/p/bamtools}

    \item[8.] 1000 Genomes Project Consortium.. 
    A map of human genome variation from population-scale sequencing.\\ 
    \textit{Nature} 7319:1061-73, 2010.

    \item[7.] \textbf{Quinlan AR} and Hall IM.
    BEDTools: A flexible framework for comparing genomic features.\\ 
    \textit{Bioinformatics}, 6:841-842, 2010.\\
    \url{code.google.com/p/bedtools}

    \item[6.] \textbf{Quinlan AR}, Clark RA, Sokolova, S, Leibowitx ML, Zhang Y, Hurles ME, Mell JC, Hall IM. 
    Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.\\
    \textit{Genome Research}, 20:623-635, 2010.\\
    \url{code.google.com/p/hydra-sv}
    
    \end{list}
    

    \textbf{2007 - 2009} \\
    \begin{list}{*}{}

    \item[5.] Sackton, TB, Kulathinal RJ, Bergman CM, Quinlan AR, Dopman E, Marth GT, Hartl DL, Clark AG. 
    Population Genomic Inferences from Sparse High-Throughput Sequencing of Two Populations of Drosophila melanogaster.\\
    \textit{Genome Biol Evol}, 1:439-455, 2009.

    \item[4.] Smith D, \textbf{Quinlan AR}$\star$, Peckham HR, \textit{et al}. 
    Rapid whole-genome mutational profiling using next-generation sequencing technologies.\\
    \textit{Genome Research}, 18:1638-1642, 2008.

    \item[3.] Hillier LW, Marth GT, \textbf{Quinlan AR}, \textit{et al}. 
    Whole Genome Sequencing and SNP Discovery for C. elegans using massively parallel sequencing-by-synthesis.\\
    \textit{Nature Methods}, 5:183-188, 2008.
    
    \item[2.] \textbf{Quinlan AR}, Stewart D, Stromberg M, Marth GT. 
    PyroBayes: Accurate quality scores for 454 Life Science pyrosequences.\\
    \textit{Nature Methods}, 5:179-181, 2008.
    
    \item[1.] \textbf{Quinlan AR}, Marth GT. 
    Primer-site SNPs mask mutations.\\
    \textit{Nature Methods}, 4:192, 2007.

    \end{list}

    

    %__________________________________________________________________________________________________________________
    % Research Support
    \section{\mysidestyle Active Research Support}

    Project Title: \textit{New algorithms and tools for large-scale genomic analysis.} \\
    PI: Aaron Quinlan \\
    Source: NIH/NHGRI (R01 HG006693-01) \\
    Period funded: 19-Apr-2012 - 31-Mar-2016
    
    Project Title: \textit{A clinical sequencing program to direct treatment of relapsed pediatric cancers.} \\
    PI: Ira Hall and Aaron Quinlan \\
    Source: UVA Health System Research Award  \\
    Period funded: 31-Apr-2013 - 30-Mar-2016
	    
    \vspace{-2mm}
    Project Title: \textit{New oncogenes and regions of genome instability in ovarian cancer.} \\
    PI: Aaron Quinlan \\
    Source: University of Virginia Fund for Excellence in Science and Technology (FEST) \\
    Period funded: 01-May-2011 - 30-Apr-2014
    
    \vspace{-2mm}
    Project Title: \textit{Defining the genomic architecture of glioblastoma for improved therapy.} \\
    PI: Aaron Quinlan \\
    Source: University of Virginia Cancer Center Pilot Fund\\
    Period funded: 01-Apr-2011 - 31-Dec-2013

    \vspace{-2mm}
    Project Title: \textit{The Role of Copy Number Variants in Type 1 Diabetes.} \\
    PI: Stephen Rich \\
    Source: NIH/NIDDK (DP3 DK085695)\\
    Period funded: 30-Sep-2009 - 30-Jun-2014

    \vspace{-2mm}
    Project Title: \textit{Expression and proteomic characterization of risk loci in type 1 diabetes.} \\
    PI: Stephen Rich \\
    Source: NIH/NIDDK (DP3 DK085678)\\
    Period funded: 25-Sep-2009 - 30-Jun-2014


    \section{\mysidestyle Completed Research Support}

    Project Title: \textit{Identification of radiation sensitivity alleles by whole exome sequencing.} \\
    PI: Pat Concannon \\
    Co-investigator: Aaron Quinlan \\
    Source: NIH/NIEHS (R21 ES020521-01) \\
    Period funded: 19-Aug-2011 - 31-Jul-2013
    
    \vspace{-2mm}
    Project Title: \textit{Carry Out Physical Characterization of Contest Samples and Development and Test Bioinformatic Methods for Scoring and Judging Contestant Entries.}\\
    PI: Dean Gaalaas\\
    Source: \\
    Period Archon Genomics X Prize : 27-Mar-2012 - 31-Dec-2012

    \vspace{-2mm}
    Project Title: \textit{Rates and patterns of recurrent structural variation in the mouse genome.}\\
    PI: Aaron Quinlan\\
    Source: NIH/NHGRI (F32 HG005197-02)\\
    Period funded: 01-Aug-2009 - 31-Dec-2010

    %__________________________________________________________________________________________________________________
    % Seminars
    \section{\mysidestyle Lectures}

    $\dagger$\textit{invited lecture}\\
    $\star$\textit{abstract selected lecture}

    $\star$ \textit{Variant calling while accounting for 
    alternate haplotypes} \\
    Genome Reference Consortium Workshop 2014; Cambridge, England; \\
    Sep 21, 2014 \\

    $\star$ \textit{How does ovarian cancer become resistant to chemotherapy?} \\
    Biology of the Genome 2014; Cold Spring Harbor, NY \\
    May 6-10, 2014

    $\dagger$ \textit{Prioritizing germline and somatic variation in studies of human disease.} \\
    Johns Hopkins School of Medicine. \\
    April 28, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Virginia Biomedical Engineering Seminar Series. \\
    April 4, 2014

    $\dagger$ \textit{Algorithms for chromosomal rearrangement detection and DNA classification.} \\
    Cold Spring Harbor Laboraties Quantitative Biology Seminar Series. \\
    March 19, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Florida Genetics Institute. \\
    March 12, 2014

    $\dagger$ \textit{Comprehensive discovery and prioritization of genetic variation in studies of human disease.} \\
    University of Utah Department of Human Genetics. \\
    February 26, 2014

    $\star$ \textit{Disease variant interpretation and prioritization with GEMINI.} \\
    Genome Informatics 2013; Cold Spring Harbor, NY \\
    October 30, 2013

    $\star$ \textit{Mining genomic feature sets and identifying significant biological relationships with BedTools2.} \\
    American Society of Human Genetics; Boston, MA \\
    October 22, 2013

    $\star$ \textit{Disease variant interpretation and prioritization with GEMINI.} \\
    Beyond the Genome 2013; San Francisco, CA \\
    October 3, 2013

    $\dagger$ \textit{Detection and characterization of complex rearrangements in 
	tumor genomes.} \\
    BioConductor 2013; Seattle, WA \\
    July 18, 2013
	
    $\star$ \textit{Exploring disease genetics among thousands of human genomes 
	with GEMINI.} \\
    SciPy 2013; Austin, TX \\
    June 26, 2013
	
    \textit{Computational Genomics.} \\
    Big Data Summit 2 at UVa; Charlottesville, VA \\
    May 14, 2013
	
    \textit{Exploring genetic variation with a tour guide.} \\
    International Stroke Genetics Consortium Meeting; Charlottesville, VA \\
    April 25, 2013

    $\star$ \textit{LUMPY: A probabilistic framework for SV discovery.} \\
    Advances in Genome Biology and Technology (AGBT); Marco Island, FL \\
    February 22, 2013

    \textit{Mining the genome.} \\
    UVa. Center for Public Health Genomics Genome Sciences Seminar Series \\
    November 28, 2012
    
    $\dagger$ \textit{Mining the structure and function of the genome.} \\
    Penn State, Dept. of Biochemistry and Molecular Biology \\
    November 12, 2012; Host: Anton Nekreutenko
    
    \textit{Exploring high-dimension genomic data.} \\
    Cold Spring Harbor Laboratories Advanced Sequencing Technologies Course \\
    October 22, 2012
    
    \textit{Towards a map of structural variation in the Exome Sequencing Project.} \\
    NHLBI Exome Sequencing Project In-Person Meeting \\
    March 28, 2012

    $\dagger$ \textit{Exploring the origin and extent of structural variation in human genomes.} \\
    Dean’s New Faculty Seminar Series, University of Virginia School of Medicine \\
    Jan 19, 2012

    \textit{ESP Structural Variation Project Group: goals, initial results, and future work.} \\
    NHLBI Exome Sequencing Project In-Person Meeting \\
    June 9, 2011
    
    $\star$ \textit{Large-Scale Characterization of SV Breakpoints in Cancer.} \\
    Keystone Symposium on The Functional Impact of Structural Variation \\
    Jan. 11, 2011

    $\star$ \textit{Efficient discovery of structural instability in repetitive regions of mammalian genomes.} \\
    Advances in Genome Biology and Technology \\
    Feb. 2009

    \textit{Approaches to rare allele discovery: More samples or more depth per sample? } \\
    1000 Genomes Analysis Meeting, Cold Spring Harbor Laboratories \\
    May 2008.
    
    %__________________________________________________________________________________________________________________
    % Academic Service
    %__________________________________________________________________________________________________________________
    \section{\mysidestyle Academic Service}

    Judge for BIMS Graduate Student Poster Session. \\
    19-Apr-2013.

    Data Management Committee. Organized by Rick Horwitz, VPR.\\
    12-Apr-2013.

    Big Data Analytics Committee. Organized by Don Brown, Systems Engineering.\\
    29-Mar-2013.

    Member of Center for Public Health Genomics Executive Committee.\\
    01-Aug-2012 - Present.
    
    \vspace{-2mm}
    Member of the Univ. of Virginia Bioinformatics Core Advisory Committee.\\
    01-Nov-2011 - Present.
    
    \vspace{-2mm}
    Served on the Univ. of Virginia Bioinformatics Core Director Search Committee. \\
    Summer 2011.
    %__________________________________________________________________________



    %__________________________________________________________________________
    % Teaching
    %__________________________________________________________________________
    \section{\mysidestyle Teaching}


    Faculty for CSHL Advanced Sequencing Technologies Course, 2009, 2010, 2011, 2012, 2013, 2014\\
    \emph{- http://meetings.cshl.edu/courses/2013/c-seqtech13.shtml}

    \vspace{-2mm}
    Faculty for the University of Washington's Center for Mendelian Genetics Workshop. August, 2014
    
    \vspace{-2mm}
    Faculty for Canadian Bioinformatics Workshops. 2012, 2013, 2014 \\
    \emph{- http://bioinformatics.ca/workshops/faculty}

    \vspace{-2mm}
    Guest Lecturer for first year Computer Science graduate student core course (CS 6190), Fall 2013. Taught by Kevin Skadron.

    \vspace{-2mm}
    Lecturer for first year graduate student core course (BIMS 6000), Fall 2013 \\
    \emph{- Genomics Lecture and practical session} - 04-Sep-2013 \\
    \emph{- Research article discussions} - 5-Sep-2013

    \vspace{-2mm}   
    Guest Lecturer for undergraduate and graduate Biomedical Engineering course (BME 4806, 7806; Prof. Brent French), \\
    \emph{April 8, 2013} \\
    \emph{April 14, 2014} \\


    Guest Lecturer for graduate course in genomics (BIOCH 5080), Spring 2013. \\
	\emph{March 20, March 22, April 29}
	
    Lecturer for first year medical student curriculum, Fall 2012. \\
	\emph{- Genomics research article discussions} - 09-Oct-2012

    \vspace{-2mm}	
    Lecturer for first year graduate student core course (BIMS 6000), Fall 2012 \\
	\emph{- Genomics Lecture and practical session} - 07-Sep-2012 \\
	\emph{- Research article discussions} - 10-Sep-2012, 11-Sep-2012
    
    \vspace{-2mm}
    Guest Lecturer for graduate course in genomics (BIOCH 5080), 15-Feb-2011   
    %__________________________________________________________________________
    
    

    %__________________________________________________________________________
    % Mentorship
    %__________________________________________________________________________    
    \section{\mysidestyle Mentorship}
    \textbf{Current} \\
    James Havrilla (Ph.D. candidate, Biochemistry and Molecular Genetics, started 2014) \\
    Phanwadee Sinthong (Undergraduate Computer Science researcher) \\

    \textbf{Former} \\
    John Kubinski (Undergraduate Biology Senior Thesis candidate)  \\
    Ryan Layer (Ph.D. candidate, Computer Science, graduated 2014) \\
    %__________________________________________________________________________
    
        
    %__________________________________________________________________________
    % Thesis Committees
    %__________________________________________________________________________    
    \section{\mysidestyle Thesis Committees}
    \textbf{Current} \\
    Michael Lindberg (Ph.D. candidate, Biochemistry and Molecular Genetics, qualified 2012)\\
    Lauren Mills (Ph.D., candidate Biochemistry and Molecular Genetics)\\
	Johnny Gan (Ph.D., candidate Systems Engineering)
    %__________________________________________________________________________
    
    

%______________________________________________________________________________
\end{resume}
\end{document}


%______________________________________________________________________________
% EOF

